stateupgrade/state_upgrade.go

```diff
@@ -0,0 +1,45 @@
+// (c) 2023 Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package stateupgrade
+
+import (
+	"math/big"
+
+	"github.com/ava-labs/subnet-evm/params"
+	"github.com/ethereum/go-ethereum/common"
+)
+
+// Configure applies the state upgrade to the state.
+func Configure(stateUpgrade *params.StateUpgrade, state StateDB) error {
```

Comment:
why this is not a method for `StateUpgrade`? to keep this same with precompile upgrades?

Reply:
Yes. I don't think we want to import the execution functionality from `params`, otherwise we may encounter the same import loop we saw with precompiles originally

Reply:
> Yes. I don't think we want to import the execution functionality from `params`, otherwise we may encounter the same import loop we saw with precompiles originally

we would not need if we move it to params :D